src/js/edu-benefits/definitions/educationProgram.js

```diff
@@ -2,31 +2,18 @@ import _ from 'lodash/fp';
 
 import { showSchoolAddress } from '../utils/helpers';
 import * as address from '../../common/schemaform/definitions/address';
-import educationTypeUISchema from '../definitions/educationType';
+import educationTypeUISchema from './educationType';
 
 export const uiSchema = {
   'ui:order': ['name', 'educationType', 'address'],
-  address: address.uiSchema(),
+  address: _.merge(address.uiSchema(), {
```

Comment:
Is the only thing this change is doing is simplifying the code? And looks like the 1990e is also using this file, so just want to make sure the changes here don't negatively impact the 1990e.